server/src/main/kotlin/suwayomi/tachidesk/manga/impl/Page.kt

```diff
@@ -134,28 +145,71 @@ object Page {
             getImageResponse(cacheSaveDir, fileName) {
                 source.getImage(tachiyomiPage)
             },
+            isDownloader,
             format,
         )
     }
 
     private suspend fun convertImageResponse(
         image: Pair<InputStream, String>,
+        isDownloader: Boolean,
         format: String? = null,
     ): Pair<InputStream, String> {
-        val imageExtension = MimeUtils.guessExtensionFromMimeType(image.second) ?: image.second.removePrefix("image/")
+        var currentImage = image.first
+        var currentMimeType = image.second
+
+        val conversions = serverConfig.serveConversions.value
+        val defaultConversion = conversions["default"]
+        val conversion = conversions[currentMimeType] ?: defaultConversion
+
+        // Apply HTTP post-process if configured (complementary with format conversion)
+        if (!isDownloader && conversion != null && ConversionUtil.isHttpPostProcess(conversion)) {
```

Comment:
I'm not quite sure I understand your intensions here, does this not mean non-url conversions are ignored on serve?

Together with the other comment is seems like on download we only support regular convert, and on serve only url-based. If so, the documentation is misleading.